### PR TITLE
SO-408 - restrict appropriate sections in seller profile

### PR DESCRIPTION
**Pull request by adohall**

### CODE REVIEW

File: app/templates/components/seller-profile/capability-and-experty.hbs [hunk 1]
```diff
@@ -12,14 +13,15 @@
...
-        {{seller-profile/edit-save
-          theme="sapphire"
-          underEdit=(mut underEdit)
-          form=form
-        }}    
-      </div>
+      {{#dds/info-box class="col-xs-18"}}
+        {{fa-icon "info-circle" prefix="fas"}} Only available to registered buyers
+      {{/dds/info-box}}
+      {{seller-profile/edit-save
```
Comment (armansa) [inline app/templates/components/seller-profile/capability-and-experty.hbs]: Why edit-save is added again on this view? It should be rendered on the top layer component.

File: app/templates/components/seller-profile/government-credential.hbs [hunk 2]
```diff
@@ -13,14 +14,15 @@
...
-        {{seller-profile/edit-save
-          theme="sapphire"
-          underEdit=(mut underEdit)
-          form=form
-        }}    
-      </div>
+      {{#dds/info-box class="col-xs-18"}}
+        {{fa-icon "info-circle" prefix="fas"}} Only available to registered buyers
+      {{/dds/info-box}}
+      {{seller-profile/edit-save
```
Comment (armansa) [inline app/templates/components/seller-profile/government-credential.hbs]: ditto

File: app/templates/seller-profile.hbs [hunk 3]
```diff
@@ -83,34 +83,48 @@
...
           {{/if}}
           {{#if this.model.seller.tags}}
-            <hr class="super-light">
-            <p><b>Business identifiers</b></p>
-            <p>
-              {{#each this.model.seller.tags as |tag|}}
-                {{dds/tag theme="sapphire" label=tag class="inline"}}
-              {{/each}}
-            </p>
+            {{#if buyerView}}
```
Comment (armansa) [inline app/templates/seller-profile.hbs]: Please use and helper instead of nested conditions

File: app/templates/seller-profile.hbs [hunk 4]
```diff
@@ -83,34 +83,48 @@
...
+              <p><b>Business identifiers</b></p>
+              <p>
+                {{#each this.model.seller.tags as |tag|}}
+                  {{dds/tag theme="sapphire" label=tag class="inline"}}
+                {{/each}}
+              </p>
+            {{/if}}
           {{/if}}
           {{#if this.model.seller.documents}}
+            {{#if buyerView}}
```
Comment (armansa) [inline app/templates/seller-profile.hbs]: Ditto

File: app/templates/seller-profile.hbs [hunk 5]
```diff
@@ -139,21 +153,19 @@
         <div>
           {{dds/top-link}}
         </div>
-      {{/if}}
 
-      <h5 class="au-display-lg" id="references-and-case-studies">
-        <b class="dark-sapphire">References and case studies</b>
-      </h5>
+        <h5 class="au-display-lg" id="references-and-case-studies">
```
Comment (armansa) [inline app/templates/seller-profile.hbs]: These codes are moved inside the if this.model.seller.documents condition, which is not correct

File: app/templates/seller-profile.hbs [hunk 6]
```diff
@@ -139,21 +153,19 @@
...
-      <div>
-        {{dds/top-link}}
-      </div>
+        <div>
+          {{dds/top-link}}
+        </div>
 
-      {{#if buyerView}}
         <h5 class="au-display-lg" id="government-credentials">
           <b class="dark-sapphire">Government credentials</b>
```
Comment (armansa) [inline app/templates/seller-profile.hbs]: ditto

File: app/templates/seller-profile.hbs [hunk 7]
```diff
@@ -163,19 +175,17 @@
...
           {{dds/top-link}}
         </div>
-      {{/if}}
 
-      <h5 class="au-display-lg" id="schemes-and-panels">
-        <b class="dark-sapphire">Schemes and panels</b>
-      </h5>
-      <br>
-      {{seller-profile/scheme-and-panel-view form=this.model.scheme_and_panel buyerView=buyerView}}
+        <h5 class="au-display-lg" id="schemes-and-panels">
```
Comment (armansa) [inline app/templates/seller-profile.hbs]: ditto

File: app/templates/components/seller-profile/capability-and-experty.hbs (hunk 1 above)
Comment (adohall) [reply app/templates/components/seller-profile/capability-and-experty.hbs]: This seemed to be the best way to achieve a side-by-side layout of info box and edit button if  the user is viewing the section, and just a save button if the user is editing. If there's a better way to achieve this, let me know and I'll change it.

File: app/templates/components/seller-profile/government-credential.hbs (hunk 2 above)
Comment (adohall) [reply app/templates/components/seller-profile/government-credential.hbs]: As above.

File: app/templates/seller-profile.hbs (hunk 3 above)
Comment (adohall) [reply app/templates/seller-profile.hbs]: Done

File: app/templates/seller-profile.hbs (hunk 4 above)
Comment (adohall) [reply app/templates/seller-profile.hbs]: Done

File: app/templates/components/seller-profile/capability-and-experty.hbs (hunk 1 above)
Comment (armansa) [reply app/templates/components/seller-profile/capability-and-experty.hbs]: Sorry I just didn't read the code well!

File: app/templates/seller-profile.hbs [hunk 8]
```diff
@@ -10,13 +10,13 @@
...
   {{dds/jump-to class="on-the-edge pull-right"
     sections=(array
-      (hash name="Licenses, accreditations and awards" anchor="licenses-accreditations")
-      (hash name="Special capabilities and expertise" anchor="special-capabilities")
-      (hash name="References and case studies" anchor="references-and-case-studies")
-      (hash name="Government credentials" anchor="government-credentials")
-      (hash name="Schemes and panels" anchor="schemes-and-panels")
-      (hash name="Team members" anchor="team-members")
-      (hash name="Promotional video" anchor="promotional-video")
+      (hash name="Licenses, accreditations and awards" anchor="licenses-accreditations" show=true)
```
Comment (armansa) [inline app/templates/seller-profile.hbs]: Here I recommend to send hide attribute instead of show.

File: app/templates/seller-profile.hbs (hunk 5 above)
Comment (adohall) [reply app/templates/seller-profile.hbs]: They're inside the "if buyerView" condition now.

File: app/templates/seller-profile.hbs (hunk 6 above)
Comment (adohall) [reply app/templates/seller-profile.hbs]: As above.

File: app/templates/seller-profile.hbs (hunk 7 above)
Comment (adohall) [reply app/templates/seller-profile.hbs]: As above.

File: app/templates/seller-profile.hbs (hunk 8 above)
Comment (adohall) [reply app/templates/seller-profile.hbs]: Done

**Pull request merged**